inc/Ui/Search.php

```diff
@@ -579,33 +581,37 @@ protected function getFulltextResultsHTML($data, $highlight)
             }
 
             $snippet = '';
-            $lastMod = '';
+            $hits = '';
+            $resultBody = [];
             $mtime = filemtime(wikiFN($id));
+            $lastMod = '<span class="lastmod">' . $lang['lastmod'] . '</span> ';
+            $lastMod .= '<time datetime="' . date_iso8601($mtime) . '" title="'.dformat($mtime).'">' . dformat($mtime, '%f') . '</time>';
             if ($cnt !== 0) {
-                $resultHeader[] = $cnt . ' ' . $lang['hits'];
+                $hits = '<span class="hits">' . $cnt . ' ' . $lang['hits'] . '</span>';
                 if ($num < FT_SNIPPET_NUMBER) { // create snippets for the first number of matches only
-                    $snippet = '<dd>' . ft_snippet($id, $highlight) . '</dd>';
-                    $lastMod = '<span class="search_results__lastmod">' . $lang['lastmod'] . ' ';
-                    $lastMod .= '<time datetime="' . date_iso8601($mtime) . '" title="'.dformat($mtime).'">' . dformat($mtime, '%f') . '</time>';
-                    $lastMod .= '</span>';
+                    $snippet = ft_snippet($id, $highlight);
                 }
                 $num++;
             }
 
-            $metaLine = '<div class="search_results__metaLine">';
-            $metaLine .= $lastMod;
-            $metaLine .= '</div>';
+            $resultBody['meta'] = $hits . $lastMod;
 
+            if ($snippet) {
```

Comment:
Why is this here at all? Could it not be added to `$resultBody['snippet']` in the same line in which the snippet gets created a few lines above?